kep/20200116-coscheduling-plugin-based-on-scheduler-framework.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Coscheduling plugin based on scheduler framework
```

Comment:
To distinguish with other coscheduling implementation, can we reword it to "Lightweight coscheduling based on back-to-back queue sorting"? And also rename this file as well.

Reply:
Done